doc/modules/model_evaluation.rst

```diff
@@ -96,17 +96,32 @@ Scoring                                Function
 'neg_mean_squared_error'               :func:`metrics.mean_squared_error`
 'neg_root_mean_squared_error'          :func:`metrics.root_mean_squared_error`
 'neg_mean_squared_log_error'           :func:`metrics.mean_squared_log_error`
-'neg_root_mean_squared_log_error'      :func:`metrics.root_mean_squared_log_error`
```

Comment:
Why remove this line? It seems like a valid scorer name:

```python
>>> import sklearn
>>> sklearn.metrics.get_scorer("neg_root_mean_squared_log_error")
make_scorer(root_mean_squared_log_error, greater_is_better=False, response_method='predict')
```

Reply:
Furthermore, it's not really related to the problem of defining scorer objects for metrics that need or benefit from passing a user settable parameter.

Reply:
My mistake as explained above in the response to @glemaitre's comment — it was missing in an older sklearn version where I ran `metrics.get_scorer_name()` used to cross-check.